docs/creating_and_using_service_accounts_for_CI.md

```diff
@@ -0,0 +1,142 @@
+# Creating and Using Service Accounts for ClusterPools in CI/Automation
+
+One of the primary use-cases of Clusterpools is to provide environments rapidly on-demand for CI with pre-configured security constraints, ICSPs, sizing, etc.  Clusterpools outsource the difficult, time consuming, expensive, and error-prone provisioning problem to ACM/Hive and provide an easy interface to acquire Clusters.  We use ClusterPools to drive our CI at scale (up to 72 clusters consumed by our integration tests per day).  
+
+But - how do you interface with the cluster hosting your ClusterPools if you use token-based identity providers like GitHub?  We recommend that you use ServiceAccounts with some custom ClusterRoles and ClusterRoleBindings.  Service accounts provide token-based authentication for CI systems to use to access ClusterPools. 
+
+## Note on ServiceAccount "Groups"
+
+If you would like to give all service accounts in a given namespace access to clusterpools/clusterpool resources, you can optionally replace the ServiceAccount `subjects` list entries found in the following sections with a 'group' as follows:
+```
+subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: 'system:serviceaccounts:__SERVICE_ACCOUNT_NAMESPACE__'
+```
+
+## Creating a Service Account and Configuring Roles
+
+The below yaml is a one-stop-shop for creating the necessary ClusterRoles, ServiceAccount, and ClusterRoleBindings for a clusterpool (after filling in the template items).  The following will create:
+
+**Custom ClusterRole**
+    - grants access to create/delete ClusterClaims (in order to claim clusters from our pools)
+
+**ClusterRoleBindings**
+    - `clusterpool-user` for access to create/delete clusterpools and claims
+    - `view` to view the project that holds our clusterpools
```

Comment:
`view` is too generic of a name in my opinion

Reply:
`view` is an official OCP-shipped ClusterRole - so we're stuck with it!